<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWnd</class>
 <widget class="QMainWindow" name="MainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>731</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Match Maker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_parent">
         <item>
          <widget class="QTreeView" name="treeView">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_table">
           <item>
            <widget class="QTableView" name="tableView">
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_filter">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Username filter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="filterLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="filterPushButton">
               <property name="text">
                <string>Filter</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="status_line_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_the_Dashboard_to_File"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_Add_user"/>
    <addaction name="action_Remove_user"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Hide_Dashboard"/>
    <addaction name="actionShow_Hide_User_List"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_the_Dashboard_to_File">
   <property name="text">
    <string>Save the Dashboard to File</string>
   </property>
  </action>
  <action name="action_Add_user">
   <property name="text">
    <string>+Add user</string>
   </property>
  </action>
  <action name="action_Remove_user">
   <property name="text">
    <string>-Remove user</string>
   </property>
  </action>
  <action name="actionShow_Hide_Dashboard">
   <property name="text">
    <string>Show /Hide Dashboard</string>
   </property>
  </action>
  <action name="actionShow_Hide_User_List">
   <property name="text">
    <string>Show / Hide User List</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
